alpha/alpha-core/src/main/java/org/apache/servicecomb/pack/alpha/core/OmegaCallback.java

```diff
@@ -17,9 +17,18 @@
 
 package org.apache.servicecomb.pack.alpha.core;
 
+import org.apache.servicecomb.pack.alpha.core.fsm.CompensateAskType;
```

Comment:
How about change the CompensateAskType to CompensateAckType;

Reply:
Ok, It's a typo

Reply:
done